### DISCUSSION TITLE
Is ExpressoTS the same as NestJS

### DISCUSSION THREAD

**Discussion by bouchtaoui-dev** [expressots/expressots#82] [General]
Hello there,

I stumbled on your project, but I wonder if what you do is the same as NestJS?
Or is it much easier to use EspressoTS than NestJS.

Thank you!

**Comment by rsaz** [comment]
Hey @bouchtaoui-dev, thanks for your question! While I haven't had the chance to use Nestjs in a professional setting, I've been a fan of ExpressoTS since day one and led all my TypeScript/JavaScript projects with it.

To get right to your question: yes, building backend applications and APIs with ExpressoTS is both quicker and easier. One of the best parts? New devs can jump into a project almost immediately, thanks to our well-organized project templates and clearly defined application development workflow.

If you're curious about performance as well, please check out our latest benchmark here: [ExpressoTS Benchmark](https://github.com/expressots/expressots/wiki/Benchmark).

I genuinely believe that ExpressoTS offers a more intuitive workflow than other tools out there. Our framework draws inspiration from industry stalwarts like .NET and Spring Boot, so you'll find familiar elements incorporated into ExpressoTS.

There's a ton more I could say, but I think my next step will be to write an article comparing ExpressoTS with other frameworks. Stay tuned!

Cheers!

**Comment by rsaz** [comment]
Closing the question assuming doubts were sorted it out.
